modules/db/src/main/java/org/mule/module/db/internal/domain/type/MetadataDbTypeManager.java

```diff
@@ -101,7 +101,16 @@ protected void initialise(DbConnection connection)
                 Number data_type = (Number) typeRecord.get(METADATA_TYPE_ID_COLUMN);
                 String type_name = (String) typeRecord.get(METADATA_TYPE_NAME_COLUMN);
 
-                ResolvedDbType resolvedDbType = new ResolvedDbType(data_type.intValue(), type_name);
+                ResolvedDbType resolvedDbType = null;
+
+                if (data_type.intValue() == Types.CLOB)
```

Comment:
I'm not sure that this change is OK. This kind of types should be being managed by StaticDbTypeManager. Did you review that class and how is being used?